pkgs/development/libraries/catboost/default.nix

```diff
@@ -79,24 +108,45 @@ stdenv.mkDerivation (finalAttrs: {
   cmakeFlags = [
     "-DCMAKE_BINARY_DIR=$out"
     "-DCMAKE_POSITION_INDEPENDENT_CODE=on"
-    "-DCATBOOST_COMPONENTS=app;libs${lib.optionalString pythonSupport ";python-package"}"
+    "-DCATBOOST_COMPONENTS=app;libs${lib.optionalString pythonSupport ";python-package"}${lib.optionalString rLibrary ";R-package"}"
```

Comment:
Python & r support look like they're mutually exclusive? Might be worth asserting.

Reply:
Added, thanks